airflow/contrib/operators/gcp_bigtable_operator.py

```diff
@@ -259,7 +263,7 @@ def _compare_column_families(self):
             self.log.error("Actual: %s", table_column_families.keys())
             return False
 
-        for key in table_column_families.keys():
+        for key in table_column_families:
```

Comment:
Can you explain this change?

Reply:
Pylint:

> consider-iterating-dictionary (C0201): 
> Consider iterating the dictionary directly instead of calling .keys()

Reply:
Och. Got it. I see that we have about 15 similar problems in the project, which we will improve in the future